### ISSUE TITLE
[FEATURE REQUEST] Start in the background

### ISSUE THREAD

**Issue by joao-carmassi** [rivafarabi/deckboard#1138]
I use the deckboard every time I turn on my Windows, but it bothers me a little that every time my PC turns on the application window opens every time, I would like to know if in any future update we could have a button to make the application start with Windows but only starts in the background, without opening any windows

**Comment by ZantsuRocks** [comment]
#24 this request exists at least for SIX YEARS and the dev don't care, even dozen of people asking for it.

**Comment by ZantsuRocks** [comment]
> This issue is stale because it has been open for 1 year with no activity.

Yes GH-Actions this issue is stale, the dev don't care about this feature, there is a lot of issues asking for the same functionality for years and all of them is ignored.

**Comment by joao-carmassi** [comment]
that's unfortunate

**Comment by rivafarabi** [comment]
Adjustment has been made in version [3.2.0](https://github.com/rivafarabi/deckboard/releases/tag/v3.2.0).

Closing this.

<img width="1582" height="1274" alt="Image" src="https://github.com/user-attachments/assets/8d5598ae-7a20-40a2-b5ef-e915faee69e5" />
